extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -134,13 +134,14 @@ public static function onGalleryBeforeProduceHTML( $data, &$out ){
 		return true;
 	}
 
-	public static function onPortableInfoboxNodeImageGetData( $title, &$ref, $caption ) {
-
+	public static function onPortableInfoboxNodeImageGetData( $data, &$ref ) {
```

Comment:
Consider changing name both for this method and the hook itself. This hook is run while extending image data in render service not while getting data from image node